composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/MySpotsView.kt

```diff
@@ -205,3 +353,66 @@ fun MySpotsView(mySpotsViewModel: MySpotsViewModel, onAddSpotClick: () -> Unit)
         )
     }
 }
+
+
+//@Preview()
+@Composable
+fun AvailableDays() {
+    val weekDays = listOf("M", "T", "W", "T", "F", "S", "S")
```

Comment:
Might make sense to create a (String, Boolean) map here instead of using two separate lists

Reply:
Had to make something similar in this PR for an example of the map:  https://github.com/proj-overdrive/cruiser/pull/32/files#diff-a4be91e8e0b6d7e4bf65124dac7895672f573a7373b57fa620eba0105ebc6628R425